### PR TITLE
Create ci.yml

### PR DESCRIPTION
**Pull request by wimjongman**
Initial GH build actions

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+#          ${{ runner.os }}-build-
+#          ${{ runner.os }}-
+    - name: Build with Maven
+      run: mvn clean verify --no-transfer-progress -U -B -Psandbox.eclipse.org -Pstrict
+
+#    - name: Archive
+#      uses: actions/upload-artifact@v2
+#      with: 
+#        name: birt-site
+#        path: build/org.eclipse.birt.p2updatesite/target/org.eclipse.birt.p2updatesite*.zip
```
Comment (ruspl-afed) [inline .github/workflows/ci.yml:62]: I believe we need to reformulate this @wimjongman

Comment (wimjongman) [reply .github/workflows/ci.yml:62]: Go ahead, Alexander. Do your magic! 🪄

Comment (ruspl-afed) [reply .github/workflows/ci.yml:62]: I'm on it @wimjongman

**Comment by ruspl-afed** [comment]
We have CI with #9 , closing this one @wimjongman

**Pull request closed without merging**